hail/src/main/scala/is/hail/utils/package.scala

```diff
@@ -215,7 +215,7 @@ package object utils extends Logging
     .filter(s => !s.isEmpty)
 
   def prettyIdentifier(str: String): String = {
-    if (str.matches("""\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*"""))
+    if (str.matches("""[_a-zA-Z]\w*"""))
```

Comment:
this is much more conservative. There is no reason to use the java identifier stuff if we don't have a good idea what's in it -- apparently "currency" characters are valid identifier starts!